schemas/logipedia.bnf

```diff
@@ -0,0 +1,40 @@
+;; Augmented BNF reprensenting the information on a web page
+;; <ppterm> defined in an auxiliary file
+
+;; / is the alternative
+;; SP is for an explicit space
+;; quantification is prefix i.e. *ALPHA for any number of characters
+;; [ x ] is the option
+
+<id>     = ("_" / ALPHA) *(ALPHA / DIGIT)
```

Comment:
Which policy should we have for the names allowed here? I have the impression that we are a little bit too restrictve.

Reply:
any suggestion is welcome

Reply:
I think we should take inspiration from lambdapi here, and have a way to escape an identifier.

Reply:
The same thing should be done for qualified name of course. I am not sure it is a good idea though. Being too restrictive is a problem I think because we won't be able to handle a lot of identifiers.

Reply:
Indeed good idea